frontend/src/scenes/billing/billingLogic.ts

```diff
@@ -69,12 +69,20 @@ export const billingLogic = kea<billingLogicType>([
         registerInstrumentationProps: true,
         setRedirectPath: true,
         setIsOnboarding: true,
+        setBillingAlert: (billingAlert: BillingAlertConfig | null) => ({ billingAlert }),
     }),
     connect({
         values: [featureFlagLogic, ['featureFlags'], preflightLogic, ['preflight']],
         actions: [userLogic, ['loadUser'], eventUsageLogic, ['reportProductUnsubscribed']],
+        logic: [lemonBannerLogic({ dismissKey: 'usage-limit-exceeded' })],
```

Comment:
You can connect the action directly so you don't need to try and find the mounted logic and just call the action